tests/test_grpc.py

```diff
@@ -148,6 +153,15 @@ def test_download_missing_file(mapdl, tmpdir):
         mapdl.download('__notafile__', target)
 
 
+@skip_launch_mapdl  # need to be able to start/stop an instance of MAPDL
+def test_grpc_custom_ip():
+    from ansys.mapdl.core import launch_mapdl
```

Comment:
Move this to the top with the rest of our imports.

Reply:
Done